client/wordpress-com.js

```diff
@@ -193,6 +193,16 @@ sections.push( {
 	secondary: true
 } );
 
+if ( config.isEnabled( 'account-recovery' ) ) {
+	sections.push( {
+		name: 'account-recovery',
+		paths: [ '/account-recovery' ],
+		module: 'account-recovery',
+		secondary: false,
+		enableLoggedOut: true,
```

Comment:
Or I could be mistaken if you don't need SSR, though I think most of our logged out pages in Calypso are.